Block/Mailchimpjs.php

```diff
@@ -44,6 +46,17 @@ public function __construct(
     public function getJsUrl()
     {
         $storeId = $this->_storeManager->getStore()->getId();
-        return $this->_helper->getJsUrl($storeId);
+
+        $url    = $this->_scopeConfig->getValue(\Ebizmarts\MailChimp\Helper\Data::XML_MAILCHIMP_JS_URL, ScopeInterface::SCOPE_STORES, $storeId);
+        $active = $this->_scopeConfig->getValue(\Ebizmarts\MailChimp\Helper\Data::XML_PATH_ACTIVE, ScopeInterface::SCOPE_STORES, $storeId);
```

Comment:
these 2 lines exceeds maximum limit of 120 characters

Reply:
anything else?